src/modality-specific-files/magnetic-resonance-imaging-data.md

```diff
@@ -1233,7 +1233,7 @@ The [`dir-<label>`](../appendices/entities.md#dir) entity is REQUIRED
 for these files.
 This entity MUST be used in addition to
 the REQUIRED `PhaseEncodingDirection` metadata field
-(see [File name structure](../common-principles.md#file-name-structure)).
```

Comment:
IS there a way to detect those "dead anchor" links?

Reply:
Until now I thought the linkchecker would find those.

Reply:
Given that this does not fail, I think that we may have to use another strategy
https://bids-specification.readthedocs.io/en/latest/common-principles.html#foo-bar